common/src/main/scala/org/apache/celeborn/common/meta/ShufflePartitionLocationInfo.scala

```diff
@@ -25,7 +25,7 @@ import scala.collection.mutable.ArrayBuffer
 
 import org.apache.celeborn.common.protocol.PartitionLocation
 
-class ShufflePartitionLocationInfo {
+class ShufflePartitionLocationInfo(val workerInfo: WorkerInfo) {
```

Comment:
Hi @RexXiong 

I suggest to save the workerInfo into the `ShufflePartitionLocationInfo`.

So that, we can find the workerInfo with `map.get(key)`

Reply:
then do you think it is necessary to use the workerUniqId as the key of map?
Is it necessary？

https://github.com/apache/celeborn/blob/74c1ec0a7fcc4d9efb26d9b96901234eb76e22cd/client/src/main/scala/org/apache/celeborn/client/LifecycleManager.scala#L65-L66

Reply:
I am wrong, ignore

Reply:
> use the workerUniqId as the key of map?

Working on this